narwhals/_arrow/utils.py

```diff
@@ -136,9 +136,8 @@ def validate_dataframe_comparand(other: Any) -> Any:
         return NotImplemented
     if isinstance(other, ArrowSeries):
         if len(other) == 1:
-            # broadcast
-            msg = "not implemented yet"  # pragma: no cover
-            raise NotImplementedError(msg)
+            pa = get_pyarrow()
+            return pa.chunked_array([[other.item()] * length])
```

Comment:
does it work to just return `other.item()`?

Reply:
Since it has len 1 and Series implements `.item` yes 🙌🏼 but I feel like I didn't get the question?!

Reply:
as in, it should be more efficient to just return `other.item()` and let pyarrow handle the broadcasting, rather than allocating a new chunkedarray here?

Reply:
>  I feel like I didn't get the question

Yep I knew there was something 🙈

> let pyarrow handle the broadcasting

From what I tried this does not work, here what I tried and the respective result

- `return other.item()` -> `TypeError: '<type>' object is not iterable`
- `return [other.item()]` -> `Tpyarrow.lib.ArrowInvalid: Column 3 named lit expected length X but got length 1`
- `return pa.chunked_array([[other.item()]])` -> `Tpyarrow.lib.ArrowInvalid: Column 3 named lit expected length X but got length 1`

Reply:
ah right, thanks! if we ever support something like `df > scalar` then we may want to reconsider and return `other.item()` here and do the broadcasting somewhere else, but as we're only using `validate_dataframe_comparand` for `with_columns`, I think this is ok 👍